pop/service/api.go

```diff
@@ -506,8 +506,8 @@ func (desc *PopDesc) Hash() []byte {
 	return hash.Sum(nil)
 }
 
-// Equal checks if the first list contains the second
-func Equal(r1, r2 *onet.Roster) bool {
+// rosterEqual checks if the first list contains the second
+func rosterEqual(r1, r2 *onet.Roster) bool {
```

Comment:
replace it with `r1.IsRotation(r2)`.

Reply:
That causes an error. But r1.Contains(r2.Publics()) doesn't, so I'll go with that.